fastlane/spec/actions_specs/push_git_tags_spec.rb

```diff
@@ -1,12 +1,36 @@
 describe Fastlane do
   describe Fastlane::FastFile do
     describe "push_git_tags" do
-      it "uses the correct comand" do
+      it "uses the correct default comand" do
         result = Fastlane::FastFile.new.parse("lane :test do
           push_git_tags
         end").runner.execute(:test)
 
-        expect(result).to eq("git push --tags")
+        expect(result).to eq("git push origin --tags")
```

Comment:
By addressing https://github.com/fastlane/fastlane/pull/6109/files#r78658241 this test should be unchanged 👍